src/jwt/utils.ts

```diff
@@ -0,0 +1,59 @@
+// @ts-expect-error: will resolve at runtime
+import { subtle } from "uncrypto";
```

Comment:
You can directly use relative imports `..`. rollup smartly creates shared chunks for us.

Reply:
I know. But how can I combine usage of relative imports with conditional exports? Is it possible to conditionally import from `crypto.node.ts` or `crypto.web.ts`? 🙂

Reply:
Ah that's a good point. Well we can also have conditional exports for jwt but this seems easiest way (the only downside that worries me is package managers hoisting behavior)